code/datums/id_trim/_id_trim.dm

```diff
@@ -28,6 +28,12 @@
 	var/big_pointer = FALSE
 	///If set, IDs with this trim will give wearers arrows of different colors when pointing
 	var/pointer_color
+	/// What honorifics, if any, will we set our wearer's name to when worn?
+	var/list/honorifics
+	/// What positions can our honorific take? To prevent names like "Peter Dr."
+	var/honorific_positions = NONE
+	/// What is our selected honorific?
+	var/chosen_honorific
```

Comment:
Aren't these singletons? I don't think these are supposed to hold state right? So the ID card should store this